proxy/tlsproxy.go

```diff
@@ -0,0 +1,129 @@
+// tlsproxy provides a TLS proxy kind of like stunnel
```

Comment:
I think this file is now unused, right?

Reply:
What do you mean?

Reply:
tlsproxy.go is in the root folder now, right?